engine/engine/src/test/script_props/spawn.script

```diff
@@ -20,5 +22,5 @@ function init(self, config)
     assert(self.quat.y == 9)
     assert(self.quat.z == 10)
     assert(self.quat.w == 11)
-    msg.post("main:/main#script", "done")
```

Comment:
we call it 3 times per collection, so some tests have never done. I added "done" counter to use tests for similiar features in one place but in different GOs and scripts